views/components/organisms/o-header.scss

```diff
@@ -0,0 +1,113 @@
+@import "../../../src/sass/variables";
+@import "../../../src/sass/common";
+header {
+	border-bottom: solid 1px color('grey');
+	box-shadow: 1px color('shadowColor') !important;
+	position: relative;
+	.nav{
+		position: relative;
+		.nav__logo {
+			@extend .float-left;
```

Comment:
I know it will be hard now. But i will encourage you to use flex or css grid in atleast 2 components

Reply:
I have changes both the product and header page and implemented CSS grid there. Hope its fine.